app/lib/search_api.rb

```diff
@@ -0,0 +1,59 @@
+require 'services'
+
+class SearchAPI
+  def initialize(params)
+    @params = params
+  end
+
+  def search
+    search_results.merge(scope_info)
+  end
+
+private
+
+  attr_reader :params
+
+  def search_results
+    Services.rummager.search(rummager_params).to_hash
+  end
+
+  # If the user is in scoped-search mode inside a manual, then return the manual
+  # we're scoping to and the unscoped results.
+  def scope_info
+    if is_scoped? && scoped_manual.present?
```

Comment:
Did you test that scoped manual search worked as expected? Also visual diff?

Reply:
Visual regression passed against the URL: https://www.gov.uk/search?filter_manual%5B%5D=%2Fguidance%2Fcapital-funding-guide&q=pre-emption

I take it this is what you are referring to?

Reply:
https://www.gov.uk/search?filter_manual%5B%5D=%2Fguidance%2Fcapital-funding-guide&q=pre-emption
https://finder-frontend-pr-313.herokuapp.com/search?filter_manual%5B%5D=%2Fguidance%2Fcapital-funding-guide&q=pre-emption

What’s up with the glossary title changing?

![screen shot 2017-10-23 at 10 08 20](https://user-images.githubusercontent.com/319055/31881064-3ccdd5a4-b7da-11e7-82bd-760835eaff60.png)